### ISSUE TITLE
Parameter "is already assigned" when calling with spread operator

### ISSUE THREAD

**Issue by eddy-geek** [microsoft/pyright#1767]
Note: if you are reporting a wrong signature of a function or a class in the standard library, then the typeshed tracker is better suited for this report: https://github.com/python/typeshed/issues.

**Describe the bug**
False positive "Parameter groupby is already assigned"
![image](https://user-images.githubusercontent.com/2772505/115051124-c6309b00-9edc-11eb-8793-cebf1a4cb01b.png)

**To Reproduce**
```python
def foo(*args, groupby='', **kw):
    bar(*args, groupby=groupby, **kw)

def bar(baz='', groupby=''):
    pass
```

**Expected behavior**
It should typecheck.

**VS Code extension or command-line**
Pylance v2021.4.1 in vscode 1.55.2 apparently using pyright 1.1.130 patched.

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is fixed in pyright 1.1.131, which I just published. It will also be included in the next release of pylance.
